fv3gfs-physics/tests/savepoint/translate/overrides/baroclinic.yaml

```diff
@@ -65,9 +65,10 @@ Microph:
   - backend: gtcuda
     cuda_no_fma: true
   - backend: gtc:gt:gpu
+    max_error: 1.1e-10
```

Comment:
Can you describe in the PR description why this needed to be relaxed, and what it was relaxed from and to (since in this file we can only see the "to")?